tests/unit/experiments/providers/simple_experiment_tests.py

```diff
@@ -63,14 +63,14 @@ def get_simple_config():
             "variants": [
                 {
                     "name":"variant_1",
-                    "size":0.1,
+                    "size": 0.1,
                 },
                 {
                     "name":"variant_2",
-                    "size":0.1,
+                    "size": 0.1,
                 },
             ],
-            "experiment_version":1,
+            "experiment_version": 1,
```

Comment:
🔕 Interesting that this slipped through, I would figure that we had a lint check in CI for this?  Appreciate the clean up!

Reply:
yeah, that surprised me too. Not sure why this slipped through. Interestingly, I accidentally fixed it because I had the wrong file open, and linting did find that same error in another file.